src/components/LoadAssets.tsx

```diff
@@ -91,7 +94,7 @@ const LoadAssets = ({ assets, fonts, children }: LoadAssetsProps) => {
 
   return (
     <NavigationContainer {...{ onStateChange, initialState }}>
-      {children}
+      <ThemeProvider theme={theme}>{children}</ThemeProvider>
```

Comment:
This might be able to be placed outside this component which might make more sense? Either way its a minor detail